src/simple-comment-icebreakers.ts

```diff
@@ -23,8 +23,8 @@ const reverseSlug = (slug: string, questions: string[]) =>
   questions.find(question => isSlugMatch(slug, question))
 
 const fetchAndStoreQuestions = () =>
-  new Promise<string[]>((resolve, reject) => {
-    const currentTimestamp =
+  new Promise<string[]>((resolve: (questions: string[]) => void, reject: (reason?: any) => void) => {
```

Comment:
House style is to avoid `any` and use `unknown`, but specifying a type in this case is not necessary and introduces complexity.